src/Microsoft.Identity.Web.Certificateless/Certificateless.cd

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's the class diagram

![image](https://user-images.githubusercontent.com/13203188/184452084-dc107d84-1678-46b5-b7ef-5e689484f55f.png)